### DISCUSSION TITLE
fixing-max_path-issues-in-gitlab/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#437] [Announcements]
# fixing-max_path-issues-in-gitlab/

In this post I show how you can enable Git's long-path support in a GitLab job to override Window's MAX_PATH limitations which can cause git clone failures

https://andrewlock.net/fixing-max_path-issues-in-gitlab/

**Comment by AnnaLewis123** [comment]
Yes,  Long Path Tool can help fix MAX_PATH issues in GitLab, especially on Windows systems where path length limitations (typically 260 characters) cause Git operations to fail.

**Comment by AnnaLepos** [comment]
If you see “Source Path Too Long” when deleting a file or folder over 258 characters, you can fix it easily with LongPathTool. It bypasses Windows’ path-length limits and lets you delete, move, or rename long-path files without errors.

**Comment by JackyFlint** [comment]
LongPathTool resolved the MAX_PATH issue so GitLab could clone without errors.
